tools-local/tasks/mobile.tasks/WasmAppBuilder/WasmAppBuilder.cs

```diff
@@ -34,6 +35,37 @@ public class WasmAppBuilder : Task
     SortedDictionary<string, Assembly>? _assemblies;
     Resolver? _resolver;
 
+    private class WasmAppConfig
+    {
+        public string assembly_root { get; set; } = "managed";
+        public int enable_debugging { get; set; } = 0;
+        public List<object> assets { get; } = new List<object>();
```

Comment:
can this be a `List<Entry>`?

Reply:
It can

Reply:
Actually it can't System.Text.Json only serializes the specified type.  This is what broke things